### ISSUE TITLE
remove testing binary coredistools.dll from product directory

### ISSUE THREAD

**Issue by rahku** [dotnet/coreclr#6591]
PR #4789 caused test build to pollute product binaries folder by copying coredistools.dll to it. Coredistools.dll is required at test time during gcstress testing. This was found during review for PR #6578

**Comment by RussKeldorph** [comment]
@rahku Is this causing a customer-facing problem, or is it just about purity of the "Product" directory?

@swaroop-sridhar FYI

**Comment by swaroop-sridhar** [comment]
@rahku: What is the recommendation for Coredistools? Should we always download the package, but only uncompress it and copy it to the bin directory when running GCStress?

**Comment by RussKeldorph** [comment]
@swaroop-sridhar If this is just about the Product directory, presumably we could unconditionally download it and unconditionally place it in coreoverlay/Core_Root directories when those are populated.

**Comment by rahku** [comment]
this is not creating a problem. But I don't think it is good idea to pollute the product binary folder with binaries required by test code.

**Comment by rahku** [comment]
yes test core_root is the right place to copy this binary

**Comment by rahku** [comment]
@RussKeldorph Gaurav asked me to assign it to you as this was initially done by your team

**Comment by AndyAyersMS** [comment]
@RussKeldorph this issue seems stale and no longer relevant.

**Comment by mikedn** [comment]
@AndyAyersMS AFAIR coredistools appears only after building/running the tests. Not sure what's the problem with this anyway, there are other files that appear in product directories that aren't really part of the product (e.g. SuperPMI files)

**Comment by AndyAyersMS** [comment]
coredistools is now placed into the test overlay via an explicit restore step (via tests\setup-stress-dependencies.cmd). It doesn't go into the product directories at all.

So I am going to close this as the issue seems to be fixed.
